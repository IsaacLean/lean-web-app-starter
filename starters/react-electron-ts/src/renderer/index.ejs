<!doctype html>
<html>
    <head>
        <meta charset="utf-8" />
        <title><%= htmlWebpackPlugin.options.title %></title>
        <meta name="viewport" content="width=device-width,initial-scale=1" />
    </head>
    <body>
        <div id="root"></div>
        We are using Node.js <span id="node-version"></span>, Chromium
        <span id="chrome-version"></span>, and Electron
        <span id="electron-version"></span>.
    </body>
</html>
