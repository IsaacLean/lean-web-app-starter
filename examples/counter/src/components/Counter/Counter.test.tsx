import renderer from 'react-test-renderer'

import Counter from '.'

test('Counter component matches snapshot', () => {
    const tree = renderer.create(<Counter />).toJSON()
    expect(tree).toMatchSnapshot()
})
