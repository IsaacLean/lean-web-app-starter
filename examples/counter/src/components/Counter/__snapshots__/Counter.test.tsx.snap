// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter component matches snapshot 1`] = `
<div
  className="align-items-center container d-flex justify-content-center h-100"
>
  <div
    className="card"
  >
    <div
      className="card-header"
    >
      Counting App
    </div>
    <div
      className="card-body d-flex flex-column align-items-center px-5"
    >
      <p
        className="lead"
      >
        Count is currently...
      </p>
      <p
        className="display-1"
      >
        0
      </p>
      <div
        className="column-gap-3 d-flex w-100"
      >
        <button
          className="btn btn-success w-50"
          onClick={[Function]}
          type="button"
        >
          <i
            className="bi bi-dash-circle"
            style={
              {
                "fontSize": "1.5rem",
              }
            }
          />
        </button>
        <button
          className="btn btn-success w-50"
          onClick={[Function]}
          type="button"
        >
          <i
            className="bi bi-plus-circle-fill"
            style={
              {
                "fontSize": "1.5rem",
              }
            }
          />
        </button>
      </div>
      <hr
        className="w-100"
      />
      <div
        className="input-group"
      >
        <input
          className="form-control"
          onChange={[Function]}
          placeholder="Force count to be..."
          type="number"
          value=""
        />
        <button
          className="btn btn-danger"
          disabled={true}
          onClick={[Function]}
          type="button"
        >
          Force!
        </button>
      </div>
      <hr
        className="w-100"
      />
      <button
        className="btn btn-primary w-100"
        onClick={[Function]}
        type="button"
      >
        Reset
      </button>
    </div>
  </div>
</div>
`;
